src/main/scala/nl.knaw.dans.easy.bag2deposit/ddm/SplitNrRewriteRule.scala

```diff
@@ -16,23 +16,60 @@
 package nl.knaw.dans.easy.bag2deposit.ddm
 
 import scala.xml.transform.RewriteRule
-import scala.xml.{ Elem, Node, Text }
+import scala.xml.{ Elem, Node, PrefixedAttribute, Text }
 
 object SplitNrRewriteRule extends RewriteRule {
 
+  val regexpMap = Seq(
+    ".*archis[^a-z]vondstmelding.*" -> "VONDSTMELDING",
+    ".*archis[^a-z]waarneming.*" -> "WAARNEMING",
+    ".*archis[^a-z]monument.*" -> "ONDERZOEK",
+    ".*archis[^a-z]onderzoek.*" -> "VONDSTMELDING",
+    ".*archis[^a-z]onderzoek.*" -> "ZAAK-IDENTIFICATIE",
```

Comment:
```suggestion
    ".*archis[^a-z]identificatie.*" -> "ZAAK-IDENTIFICATIE",
```
??

Reply:
I must have been distracted somehow. I now also see identifiers like `Archis2-...`. What to do with them?

Reply:
https://easy.dans.knaw.nl/schemas/vocab/identifier-type.xsd says they are all in Archis2, except for the zaak-identificatie. so, let's treat them as equal